ZupIT/ritchie-cli
# feat: flag delete formula

### Description
Adding flag support to `rit delete formula` command. Some of the refactoring done as well in this PR:

- Added tests to support flags and changed tests to work with a tmp directory instead of only mocks.

Close #716 

### How to verify it
- Run `rit delete formula --workspace=<workspace name> --formula=<rit test formula>` and check if has been deleted succesfully.
- Run `rit delete formula` and check if the delete confirmation works.


### Changelog
Update delete formula for input flags
